packages/react/src/components/table/table.react-template.tsx

```diff
@@ -2,24 +2,50 @@ import { Table } from "dso-toolkit";
 import * as React from "react";
 import { clsx } from "clsx";
 
-import { DsoTable } from "../../components";
+import { DsoIcon, DsoTable } from "../../components";
 import { ComponentImplementation } from "../../templates";
+import { TableSorting } from "dso-toolkit/dist/components/table/table.models";
 
 export const reactTable: ComponentImplementation<Table<JSX.Element>> = {
   component: "table",
   implementation: "react",
   template: () =>
     function tableTemplate({ noModal, content, role, verticalLines }) {
+      function getAriaRoledescription(sorting: TableSorting): string | undefined {
+        if (sorting === "ascending") {
+          return "sorteer oplopend knop";
+        }
+
+        if (sorting === "descending") {
+          return "sorteer aflopend knop";
+        }
+
+        return undefined;
+      }
+
       return (
         <DsoTable noModal={noModal}>
           <table className={clsx("table", { ["dso-table-vertical-lines"]: verticalLines })} role={role}>
             <caption className="sr-only">{content.caption}</caption>
 
             <thead>
               <tr>
-                {content.head.map((col) => (
-                  <th scope="col" key={col} dangerouslySetInnerHTML={{ __html: col }}></th>
-                ))}
+                {content.head.map((col, cellIndex) =>
+                  "sortable" in col ? (
+                    <th scope="col" key={cellIndex}>
+                      <button
+                        type="button"
+                        aria-roledescription={getAriaRoledescription(col.sorting)}
+                        className={clsx("dso-tertiary dso-sort", { "dso-sort-active": !!col.sorting })}
+                      >
+                        <span dangerouslySetInnerHTML={{ __html: col.label }}></span>
```

Comment:
Kan `col.label` (nog steeds) HTML bevatten?

Reply:
En zo ja, zijn dat representatieve labels?

Reply:
In de Lit templates houd je geen rekening met HTML: Daar moet dan iets met de `unsafeHTML` directive worden gedaan, of `dangerouslySetInnerHTML` kan hier weg.

Reply:
Interessante vraag, ik heb niks veranderd aan het type wat in `col.label` mag. Maar dit is ALTIJD al `string` geweest. Dus waarom hier dan een `dangerouslySetInnerHTML` stond was misschien al best overkill. Zal ik hem maar weg halen?

Reply:
Klinkt goed!